dev-docs/lucene-upgrade.md

```diff
@@ -43,7 +43,11 @@ gradlew --write-locks
 ### `solr/licenses` update
 
 ```
+git rm solr/licenses/lucene-*-9.3.0.jar.sha1
```

Comment:
this step shouldn't be necessary.

If you `git add solr/licenses` it should notice the files "moved"

Reply:
ah, `git add solr/licenses` vs. `git add solr/licenses/lucene-*-9.4.0.jar.sha1` makes a difference, nice!